package.json

```diff
@@ -61,18 +61,27 @@
     "@types/lodash.merge": "^4.6.6",
     "@types/lodash.mergewith": "^4.6.6",
     "@types/node": "^10.17.26",
+    "@typescript-eslint/eslint-plugin": "5.2.0",
+    "@typescript-eslint/parser": "5.2.0",
     "benchmark": "^2.1.4",
     "codecov": "^3.3.0",
-    "graphql": "^15.1.0",
+    "eslint": "^8.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.1.1",
+    "eslint-plugin-standard": "^5.0.0",
+     "graphql": "^16.0.0",
     "jest": "^24.7.1",
     "lint-staged": "^8.1.5",
-    "prettier": "^1.19.1",
+    "prettier": "^2.4.1",
     "ts-jest": "^24.0.2",
     "ts-node": "^8.0.3",
     "tslint": "^5.15.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-plugin-prettier": "^2.0.1",
```

Comment:
Will `tslint` dependencies still be needed even if we move to `eslint`?

Reply:
Nice catch! Removed!

Reply:
My bad :) I forgot to push

Reply:
Could you take a look if it is good for you? @oporkka